### PR TITLE
[test-case] add switch scanner case

### PR DESCRIPTION
**Pull request by jwangyangls**
Signed-off-by: Yogi_Wang <yawang@vmware.com>
https://jenkins11.svc.eng.vmware.com/job/harbor_nightly_result_publisher/7493/robot/report/log.html
https://jenkins11.svc.eng.vmware.com/job/harbor_nightly_result_publisher/7495/robot/report/log.html

### CODE REVIEW

File: tests/resources/Harbor-Pages/Vulnerability.robot [hunk 1]
```diff
@@ -91,3 +91,8 @@ Trivy Is Immutable Scanner
     Retry Double Keywords When Error  Retry Element Click  ${scanner_action_xpath}  Retry Wait Until Page Contains Element  ${delete_scanner_action_xpath}
     Retry Double Keywords When Error  Retry Element Click  ${delete_scanner_action_xpath}  Retry Wait Until Page Contains Element  ${delete_scanner_confirm_btn}
     Retry Double Keywords When Error  Retry Element Click  ${delete_scanner_confirm_btn}  Retry Wait Until Page Contains Element  ${immutable_trivy_msg_xpath} 
+
+Set Default Scanner
+    [Arguments]  ${select_radio_xpath}
```
Comment (AllForNothing) [inline tests/resources/Harbor-Pages/Vulnerability.robot:96]: Is this param   needed?

File: tests/robot-cases/Group1-Nightly/Trivy.robot [hunk 2]
```diff
@@ -152,4 +152,15 @@ Test Case - Verfiy Project Level CVE Whitelist
...
\ No newline at end of file
+    Body Of Verfiy Project Level CVE Whitelist By Quick Way of Add System  goharbor/harbor-portal  2cb6a1c24dd6b88f11fd44ccc6560cb7be969f8ac5f752802c99cae6bcd592bb  CVE-2019-19317\nCVE-2019-19646 \nCVE-2019-5188 \nCVE-2019-20387 \nCVE-2019-17498 \nCVE-2019-20372 \nCVE-2019-19244 \nCVE-2019-19603 \nCVE-2019-19880 \nCVE-2019-19923 \nCVE-2019-19925 \nCVE-2019-19926 \nCVE-2019-19959 \nCVE-2019-20218 \nCVE-2019-19232 \nCVE-2019-19234 \nCVE-2019-19645 \nCVE-2019-18276
+
+Test Case - Switch Scanner
+    Init Chrome Driver
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    Switch To Scanners Page
+    Set Default Scanner  //clr-dg-row[contains(.,'Clair')]//clr-radio-wrapper/label
+    Logout Harbor
```
Comment (AllForNothing) [inline tests/robot-cases/Group1-Nightly/Trivy.robot:162]: Why need to logout?

File: tests/robot-cases/Group1-Nightly/Trivy.robot [hunk 3]
```diff
@@ -152,4 +152,15 @@ Test Case - Verfiy Project Level CVE Whitelist
...
 
 Test Case - Verfiy Project Level CVE Whitelist By Quick Way of Add System
-    Body Of Verfiy Project Level CVE Whitelist By Quick Way of Add System  goharbor/harbor-portal  2cb6a1c24dd6b88f11fd44ccc6560cb7be969f8ac5f752802c99cae6bcd592bb  CVE-2019-19317\nCVE-2019-19646 \nCVE-2019-5188 \nCVE-2019-20387 \nCVE-2019-17498 \nCVE-2019-20372 \nCVE-2019-19244 \nCVE-2019-19603 \nCVE-2019-19880 \nCVE-2019-19923 \nCVE-2019-19925 \nCVE-2019-19926 \nCVE-2019-19959 \nCVE-2019-20218 \nCVE-2019-19232 \nCVE-2019-19234 \nCVE-2019-19645 \nCVE-2019-18276
\ No newline at end of file
+    Body Of Verfiy Project Level CVE Whitelist By Quick Way of Add System  goharbor/harbor-portal  2cb6a1c24dd6b88f11fd44ccc6560cb7be969f8ac5f752802c99cae6bcd592bb  CVE-2019-19317\nCVE-2019-19646 \nCVE-2019-5188 \nCVE-2019-20387 \nCVE-2019-17498 \nCVE-2019-20372 \nCVE-2019-19244 \nCVE-2019-19603 \nCVE-2019-19880 \nCVE-2019-19923 \nCVE-2019-19925 \nCVE-2019-19926 \nCVE-2019-19959 \nCVE-2019-20218 \nCVE-2019-19232 \nCVE-2019-19234 \nCVE-2019-19645 \nCVE-2019-18276
+
+Test Case - Switch Scanner
+    Init Chrome Driver
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
```
Comment (danfengliu) [inline tests/robot-cases/Group1-Nightly/Trivy.robot:159]: After sign in harbor, Trviy should be default scanner.

File: tests/robot-cases/Group1-Nightly/Trivy.robot [hunk 4]
```diff
@@ -152,4 +152,15 @@ Test Case - Verfiy Project Level CVE Whitelist
...
+    Switch To Scanners Page
+    Set Default Scanner  //clr-dg-row[contains(.,'Clair')]//clr-radio-wrapper/label
```
Comment (danfengliu) [inline tests/robot-cases/Group1-Nightly/Trivy.robot:161]: xpath should be set to a variable in some element file.

File: tests/robot-cases/Group1-Nightly/Trivy.robot [hunk 5]
```diff
@@ -152,4 +152,15 @@ Test Case - Verfiy Project Level CVE Whitelist
...
+    Logout Harbor
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    Switch To Scanners Page
+    Clair Is Immutable Scanner
```
Comment (danfengliu) [inline tests/robot-cases/Group1-Nightly/Trivy.robot:165]: 1. Check scanned image before switch scanner, should be no scan report;
2. Scan it again, there should be a scan report;

File: tests/robot-cases/Group1-Nightly/Upgrade.robot [hunk 6]
```diff
@@ -27,7 +27,7 @@ Test Case - Manage project publicity
     Body Of Manage project publicity
 
 Test Case - Scan A Tag In The Repo
-    Body Of Scan A Tag In The Repo Use Trivy
+    Body Of Scan A Tag In The Repo  vmware/photon  1.0
```
Comment (danfengliu) [inline tests/robot-cases/Group1-Nightly/Upgrade.robot:30]: Replace line 30 with "Body Of Scan A Tag In The Repo  hello-world  latest"

**Pull request merged**